misc/alert_rules.json

```diff
@@ -1004,5 +1004,17 @@
     "rule": "applications.app_type = \"zfs\" && application_metrics.metric like \"pool_%_%_errors\" && application_metrics.value > \"0\"",
     "name": "ZFS: pool with errors present",
     "severity": "critical"
+  },
+  {
+    "rule": "storage.storage_perc >= 85 && storage.storage_perc < 90",
```

Comment:
There seems to already be some rules.

Why are none of them using storage.storage_warn?

Reply:
> There seems to already be some rules.
> 
> Why are none of them using storage.storage_warn?

storage.storage_warn tends to be useless in lots of cases.

Reply:
But aye. Good to have more options.

Added it in also removed these from default.

Reply:
Actually some one easily just yank the / bit if they don't want it. So just pulling that and leaving the warning based one.

hmm... that said pondering the idea if it could be a handy default, but I honestly can recall a time I've seen that sanely set